python/mujoco/rollout.cc

```diff
@@ -158,6 +161,47 @@ void _unsafe_rollout(std::vector<const mjModel*>& m, mjData* d, int nroll, int n
   }
 }
 
+// C-style threaded version of _unsafe_rollout
+static ThreadPool* pool = nullptr;
```

Comment:
I used a raw pointer here instead of a smart pointer because the [style guide states](https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables) that objects with static duration must be trivially destructible. My understanding is that smart pointers are not trivially destructible.